src/examples/custom-node/index.ts

```diff
@@ -0,0 +1,19 @@
+import { code, board } from "@google-labs/breadboard";
+import { merMake } from "../../util/merMake.js";
+
+const greet = code<{ name: string }>(({ name }) => {
```

Comment:
I can think of two further `code` examples that beginners might find useful: defining the code node inside the board, and creating a "one-off" anonymous code node.

Using the `create-breadboard` vanilla-ts sample as an example:
```
const myBoard = board<InputValues, OutputValues>(({}, { output }) => {
  const inputNodeOne = base.input({
    $id: "inputOne",
    schema: {
      type: "object",
      properties: {
        greet: { type: "string" },
      },
    },
  });
  const inputNodeTwo = base.input({
    $id: "inputTwo",
    schema: {
      type: "object",
      properties: {
        subject: { type: "string" },
      },
    },
  });

  const concat = code(({ base, toConcat }) => {
    const concatenated = (base as string).concat(toConcat as string);
    return { concatenated };
  })(); // <----- defining the `code` node inside the board

  inputNodeOne.greet.as("base").to(concat);
  inputNodeTwo.subject.as("toConcat").to(concat);

  const reversed = concat.concatenated.as("text").to(code(({ text }) => {
    const reversed = (text as string).split("").reverse().join("");
    return { reversed };
  })()); // <----- "one-off" anonymous `code` node

  return output({
    originalGreet: inputNodeOne.greet,
    originalSubject: inputNodeTwo.subject,
    concatenated: concat.concatenated,
    reversed: reversed
  });
});
```